lib/kubernetes-deploy/kubectl.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'kubeclient'
+
+module KubernetesDeploy
```

Comment:
Yea, it would make sense in another PR but I didn't want to open that without gathering opinions on it first.